css/inner-page.css

```diff
@@ -71,6 +60,7 @@ h3[id] {
     font-size: 24px;
     line-height: 32px;
     margin-bottom: var(--space-unit);
+    hyphens: auto;
```

Comment:
We need something more sustainable, as this way, we run a risk of adding `hyphens: auto` to more and more elements.

For example, we could add it to `html` instead, then remove it from elements where we don’t want the behavior (like in the nav).

Please add a TODO that we review use of `hyphens`.

Reply:
done, will check about what you suggested